src/www/containers/teamspage/TeamsPage.jsx

```diff
@@ -103,6 +103,12 @@ export default connect(
                 categories={categories}
                 open={this.state.isBlurred}
                 onClose={() => this.setState({ isBlurred: false })}
+                checkedCategories={[]}
+                title={t("createteam.createteam")}
+                button={t("submit")}
+                method={"post"}
+                action={"api/teams/create"}
```

Comment:
Add a leading slash here to be safe
```suggestion
                action={"/api/teams/create"}
```